install.sh

```diff
@@ -24,10 +24,10 @@ CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 ARCH="${ARCH/armv8l/armv7l}"
 
 # BOOTSTRAP_PACKAGES cannot depend on crew_profile_base for their core operations (completion scripts are fine)
-BOOTSTRAP_PACKAGES="musl_zstd pixz ca_certificates git gmp ncurses xxhash lz4 popt libyaml openssl zstd gcc rsync ruby"
-
-# Add musl bin to path
-PATH=/usr/local/share/musl/bin:$PATH
+# rsync requires openssl xxhash lz4 popt
+BOOTSTRAP_PACKAGES="zstd pixz ca_certificates ruby rsync openssl xxhash lz4 popt"
+# i686 requires gcc and openssl
```

Comment:
```
# i686 requires gcc and openssl
```
What for? (i.e. why does i686 *need* these packages when bootstrapping)

Reply:
i686 installs have such an old system they have an older libgcc and libssl than what our packages are built against.

Reply:
So does linking fail for ruby, git and ca_certificates?

Reply:
(i.e. before we install core packages)

Reply:
I don't understand the question. Apps complain about the older libraries if only those are available, and then the apps crash. This is the case for ruby for sure. Git is no longer an issue.